resources/model/OpenStudio.idd

```diff
@@ -34096,6 +34096,41 @@ OS:UtilityCost:Computation,
 
 \group OpenStudio Output Requests
 
+OS:Output:Schedules,
+       \memo Produces a condensed reporting that illustrates the full range of schedule values in
+       \memo the eio output file. In the style of input: DaySchedule,  WeekSchedule, and
+       \memo Annual Schedule.
+       \format singleLine
+       \unique-object
+  A1,  \field Handle
+       \type handle
+       \required-field
+  A2;  \field Key Field
+       \required-field
+       \type choice
+       \key Hourly
+       \key Timestep
+
+OS:Output:Constructions,
+       \memo Adds a report to the eio output file which shows details for each construction,
+       \memo including overall properties, a list of material layers, and calculated results
+       \memo related to conduction transfer functions.
+       \format singleLine
+       \unique-object
+  A1,  \field Handle
+       \type handle
+       \required-field
+  A2,  \field Constructions
+       \type choice
+       \required-field
+       \key No
+       \key Yes
+  A3;  \field Materials
+       \type choice
+       \required-field
+       \key No
+       \key Yes
```

Comment:
Thanks a lot for using a sane IDD object unlike the E+ object which is rather weird.

For reference the E+ object is like this:

```
Output:Constructions,
        \memo Adds a report to the eio output file which shows details for each construction,
        \memo including overall properties, a list of material layers, and calculated results
        \memo related to conduction transfer functions.
  \format singleLine
   A1,  \field Details Typte 1
   \type choice
   \key Constructions
   \key Materials
   A2;  \field Details Type 2
   \type choice
   \key Constructions
   \key Materials
```

Reply:
Maybe `Report For Constructions` would be clearer. But I don't really care.